Aquality.Selenium.Core/tests/Aquality.Selenium.Core.Tests/Applications/WindowsApp/ApplicationManager.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Aquality.Selenium.Core.Applications;
+using Aquality.Selenium.Core.Configurations;
+using Aquality.Selenium.Core.Utilities;
+using Microsoft.Extensions.DependencyInjection;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Service;
+using System;
+using System.Threading;
+
+namespace Aquality.Selenium.Core.Tests.Applications.WindowsApp
+{
+    public static class ApplicationManager
```

Comment:
am I right we will have similar Managers inside of each particular library (selenium,appium and etc.)?

Reply:
in fact, they could be different, depending on what is needed in the library.
from my current implementation, App Manager has a responsibility of configuring dependencies and getting them from the container; another responsibility is getting an application (which is a dependency also, so might be the same responsibility).
But 
**1) it's used to be a static class, there's no correct way to inherit common part**
2) common part could include basic interface, while what we need to get from App Manager is a full interface; cast is not an option, two properties with the same value is not a good option; so the only acceptable option for common part is to be generic - like BaseAppManager<T> where T: class, IApplication  - You can imagine how complicated it would become, so I doubt the we need it right now. But we can discuss this :)
Of course we can move factories also to this library, but in fact I'm not sure that we should do it...

Reply:
forgot all said above, I've already implemented it =) Please take a look at #8